template.yaml

```diff
@@ -11,24 +11,38 @@ Parameters:
     Description: 'Endpoint to retrieve member list for a particular Synapse team'
     Type: String
 
+Mappings:
+  Accounts:
+    '237179673806':
+      BudgetRulesFileName: 'scipoolprod'
+    '465877038949':
+      BudgetRulesFileName: 'scipooldev'
+
 Resources:
   BudgetMakerFunction:
     Type: AWS::Serverless::Function
     Properties:
-      CodeUri: budget/
-      Handler: app.lambda_handler
-      Runtime: python3.6
+      CodeUri: .
+      Handler: budget.app.lambda_handler
+      Runtime: python3.7
       Role: !GetAtt BudgetMakerFunctionRole.Arn
       Environment:
         Variables:
           NOTIFICATION_TOPIC_ARN: !Ref BudgetMakerNotificationTopic
           SYNAPSE_TEAM_MEMBER_LIST_ENDPOINT: !Ref SynapseTeamMemberListEndpoint
           AWS_ACCOUNT_ID: !Ref 'AWS::AccountId'
+          BUDGET_RULES_FILE_PATH: !Sub
+            - 'budget/config/${FileName}.yaml'
+            - FileName: !FindInMap
```

Comment:
Interesting, @zaro0508  -- why is it okay to include the configuration in synapse-login-scipool but not in this repository? Should that one be refactored also?

Reply:
synapse-login-scipool is implemented to pass in configuration files.  We initially set it up so that we define the config files in synapse-login-aws-infra  and passed those configurations to synapse-login-scipool however due to a AWS beanstalk bug not reading json files correctly[1] we had to implement a workaround.  Packaging and referencing config files in  synapse-login-scipool is the workaround.  

[1] https://sagebionetworks.jira.com/browse/SC-101?jql=project%20%3D%20SC%20AND%20text%20~%20%22beanstalk%22%20ORDER%20BY%20created%20DESC

Reply:
@zaro0508 ok, will look at doing this -- to clarify, and to avoid too much PR back and forth, do you think configurations files actually belong in the infra repo rather than say, S3?

Reply:
well, yes.  we want the configs to be in a repo so we can reap all the benefits that git and github provides.